nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/pom.xml

```diff
@@ -39,5 +39,21 @@
             <version>1.16.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-record-utils</artifactId>
+            <version>1.15.0-SNAPSHOT</version>
```

Comment:
This version needs to be updated to `1.16.0-SNAPSHOT` to match the current main branch.